src/System.Net.Http/src/System/Net/Http/Managed/ManagedHandler.cs

```diff
@@ -239,6 +239,36 @@ public SslProtocols SslProtocols
             }
         }
 
+        public TimeSpan ConnectionTimeout
+        {
+            get => _settings._connectionTimeout;
+            set
+            {
+                if (value < TimeSpan.Zero && value != Timeout.InfiniteTimeSpan)
+                {
+                    throw new ArgumentOutOfRangeException(nameof(value));
+                }
+
+                CheckDisposedOrStarted();
+                _settings._connectionTimeout = value;
+            }
+        }
+
+        public TimeSpan ConnectionIdleTimeout
```

Comment:
These are new public APIs on the internal ManagedHandler. How are we supposed to access them? Do you plan on making ManagedHandler public for 2.1?

Reply:
See #26895

Reply:
> Do you plan on making ManagedHandler public for 2.1?

Yes. https://github.com/dotnet/corefx/issues/26895